src/pages/category/Index.vue

```diff
@@ -1,43 +1,86 @@
  <template>
   <div>
-    <a-row :gutter="24">
+    <a-row :gutter="24" style>
       <a-col :span="6">
-        <a-input-search style="margin-bottom: 8px" placeholder="Search" @change="onChange" />
-        <a-tree
-          :expanded-keys="expandedKeys"
-          :auto-expand-parent="autoExpandParent"
-          :show-line="true"
-          :tree-data="categories"
-          :replaceFields="{children:'children', title:'name', key:'id' }"
-          @expand="onExpand"
+        <a-card title="分类列表" style="height:calc(100vh - 164px)">
+          <a slot="extra" href="#">新增</a>
+          <a-tree
+            :tree-data="categories"
+            :replaceFields="{children:'children', title:'name', key:'id' }"
+            autoExpandParent
+          >
+          </a-tree>
+        </a-card>
+      </a-col>
+      <a-col :span="12" style="margin-top:50px;">
+        <a-form
+          :form="editModel"
+          :label-col="{ span: 5 }"
+          :wrapper-col="{ span: 12 }"
+          @submit="createOrUpdateCategory"
         >
-          <template slot="title" slot-scope="{ title }">
-            <span v-if="title.indexOf(searchValue) > -1">
-              {{ title.substr(0, title.indexOf(searchValue)) }}
-              <span
-                style="color: #f50"
-              >{{ searchValue }}</span>
-              {{ title.substr(title.indexOf(searchValue) + searchValue.length) }}
-            </span>
-            <span v-else>{{ title }}</span>
-          </template>
-        </a-tree>
+          <a-form-item label="分类名称">
+            <a-input v-model="editModel.name" />
+          </a-form-item>
+          <a-form-item label="上级分类">
+            <a-select v-model="editModel.parentCategoryId" placeholder="无上级分类" allowClear>
+              <a-select-option
+                v-for="category in allCategories"
+                :key="category.id"
+                :value="category.id"
+              >{{ category.name }}</a-select-option>
+            </a-select>
```

Comment:
Sorry for that I probably haven't got the real problem point about this `select` control that you have mentioned in your live-show.
Just for confirm, is that mean the when trigger the clear for the select, the `placeholder` has shown but `parentCategoryId` cannot automatically set to `null`?
If 'YES', I haven't use AntD ever, but a suggestion is to try to change the `placeholder` as a vue slot for `<a-select>` rather than an attribute string. and in the slot, give a customized `option` to the `select` with value `null`.
Maybe that will work for u.

BTW: Just a suggestion to leave a TODO comment inline of related code when there have any pending issue hasn't be solved.

Reply:
> Sorry for that I probably haven't got the real problem point about this `select` control that you have mentioned in your live-show.
> Just for confirm, is that mean the when trigger the clear for the select, the `placeholder` has shown but `parentCategoryId` cannot automatically set to `null`?
> If 'YES', I haven't use AntD ever, but a suggestion is to try to change the `placeholder` as a vue slot for `<a-select>` rather than an attribute string. and in the slot, give a customized `option` to the `select` with value `null`.
> Maybe that will work for u.
> 
> BTW: Just a suggestion to leave a TODO comment inline of related code when there have any pending issue hasn't be solved.

I was tried vue solt for placeholder, but it still can not work.